eng/Build.props

```diff
@@ -102,6 +104,7 @@
         <ProjectToExclude Condition=" '$(BuildNative)' != 'true'" Include="@(NativeProjects)" />
 
         <NodeJsProjects Include="
+                          $(RepoRoot)src\Components\Web.JS\Microsoft.AspNetCore.Components.Web.JS.npmproj;
                           $(RepoRoot)src\SignalR\**\*.npmproj;
```

Comment:
Suggest moving the Web.JS project down a line because that project depends on the SignalR ones

Reply:
The order should be resolved by MSBuild. the projects comes first alphabetically 😄